RELEASING.md

```diff
@@ -20,7 +20,9 @@ Uploading a collection to the Galaxy
 ------------------------------------
 
 1. Prepare a tarball of the pulp_insaller collection (from inside the root of the collection, run `ansible-galaxy collection build`).\
-   **NOTE**: ansible-galaxy>=2.9 is reqiured.
+   **NOTE**: ansible-galaxy>=2.9 is required.\
+   **NOTE:** the tarball will include every untracked file in the repo unless you git clean, even
+ignored files. You may want to just clone the repo to another temporary dir instead.
```

Comment:
Just to keep the option in the loop: in #387 this problem is solved by copying all _wanted_ files into a build directory and creating the collection there.
This has proven to work well in squeezer, foreman-ansible-modules and forman-operations-collection.

Reply:
Thank you!